web/controllers/questionnaire_controller.ex

```diff
@@ -21,14 +22,26 @@ defmodule Ask.QuestionnaireController do
     QuestionnaireAction
   }
 
+  plug :assign_project when action in [
+    :index,
+    :show,
+    :export_zip,
+    :start_simulation,
+    :sync_simulation,
+    :get_last_simulation_response,
```

Comment:
It could be interesting to extract these 3 methods into a QuestionnaireSimulationController.